Do not show step text for environment_variable action.

thii opened the pull request:
When running in Fastlane.swift, this action prints out lots of
unnecessary step text for each environment variable being set.

thii committed:
Do not show step text for environment_variable action.

When running in Fastlane.swift, this action prints out lots of
unnecessary step text for each environment variable being set.

janpio commented:
Could you please add an example of the "unnecessary step text for each environment variable being set." to your issue text?

thii commented:
@janpio Here is a sample Fastfile.swift: 

```swift
import Foundation

class Fastfile: LaneFile {
    func testLane() {
        environmentVariable(set: [
            "FASTLANE_SKIP_UPDATE_CHECK": "YES",
        ])
    }
}
```

And here is the output:
```
[✔] 🚀
[23:22:27]: loading manifest: /Users/admin/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/fastlane-2.77.1/fastlane/swift/upgrade_manifest.json
[23:22:27]: It looks like your `FastlaneSwiftRunner` project is not up-to-date
[23:22:27]: If you don't update it, fastlane could fail
[23:22:27]: We can try to automatically update it for you, usually this works 🎈 🐐
[23:22:27]: Should we try to upgrade just your `FastlaneSwiftRunner` project? (y/n)
y
[23:22:31]: Updated your FastlaneSwiftRunner project with the newest runner code
[23:22:31]: $ xcodebuild -list -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj
[23:22:32]: $ xcodebuild -showBuildSettings -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj
[23:22:33]: $ set -o pipefail && xcodebuild -scheme FastlaneRunner -project ./fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj -destination 'generic/platform=macOS' | tee /Users/admin/Library/Logs/gym/App-FastlaneRunner.log | xcpretty
[23:22:33]: FastlaneSwiftRunner project is up-to-date
[23:22:34]: $ ./fastlane/FastlaneRunner lane test > /dev/null
[23:22:34]: ----------------------------------
[23:22:34]: --- Step: environment_variable ---
[23:22:34]: ----------------------------------

+------+----------------------+-------------+
|             fastlane summary              |
+------+----------------------+-------------+
| Step | Action               | Time (in s) |
+------+----------------------+-------------+
| 1    | environment_variable | 0           |
+------+----------------------+-------------+

[23:22:34]: fastlane.tools finished successfully 🎉
[23:22:34]: We updated your FastlaneRunner project during this run to make it compatible with your current version of fastlane.
[23:22:34]: Please make sure to check the changes into source control.
```

The step title `--- Step: environment_variable ---` is unnecessary as the `enviroment_variable` action only serves as a bridging to Ruby. Setting environment variables doesn't print out any log in the Ruby version of fastlane.

janpio commented:
Ah, now I understand - makes sense to me. Let's see if our Swift master @taquitos agrees ;)

taquitos reviewed (approved):
I agree 💯 

🎈:goat:
